packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/guides/Migration.tool.md

```diff
@@ -0,0 +1,506 @@
+
+```diff
+- browser.click('#element'); // Old v4 click. This has been removed.
++ $('#element').click(); // New v6 click. The action is called on the element instance.
+```
+
+Here are a few of the most common scenarios we encountered:
+
+```diff
+// Clicks
+- browser.click('#element');
++ $('#element').click();
+
+// Set Value
+- browser.setValue('input', 'value');
++ $('input').setValue('value');
+
+// Add Value
+- browser.addValue('input', 'value');
++ $('input').addValue('value');
+
+// Wait For Visible
+- browser.waitForVisible('#element');
++ $('#element').waitForDisplayed();
+
+// Move To
+- browser.moveToObject('#element', 0, 10);
++ $('#element').moveTo({ xOffset: 0, yOffset: 10 });
+```
+
+### Dropping Support For Node 8
+
+We're dropping support for node 8 as part of the transition to terra-functional-testing. Node 8 reached [end of life status](https://github.com/nodejs/Release/blob/master/README.md#end-of-life-releases) December 31st, 2019. Teams still on node 8 will need to upgrade to node 10.
+
+We plan to start upgrading our node version on a more routine cadence following this transition. Our range of node version support is restricted by our transitive dependency on [fibers](https://github.com/laverdet/node-fibers) which is necessary for running our WebDriverIO testing environment [synchronously](https://v6.webdriver.io/docs/sync-vs-async.html).
+
+### Assertion Framework
+
+We've removed the [chai assertion framework](https://www.chaijs.com/) and opted into using the built in [assertion library](https://v6.webdriver.io/docs/assertion.html) that ships with WebDriverIO. This new assertion framework closely resembles the testing environment most of us are used to in Jest, because it is [Jest's expect library](https://jestjs.io/docs/en/expect). This should help align testing between Jest and WebDriverIO.
+
+A few common changes look something like this:
+
+```diff
+- expect(true).to.be.true; // Chai assertion
++ expect(true).toBe(true); // Expect/Jest assertion.
+
+- expect(true).to.be.false; // Chai assertion
++ expect(true).toBe(false); // Expect/Jest assertion.
+
+- expect(1).to.equal(1); // Chai assertion
++ expect(1).toEqual(1); // Expect/Jest assertion.
+```
+
+WebDriverIO provides a collection of [matchers](https://v6.webdriver.io/docs/api/expect.html) that can be used for assertions in your tests.
+
+### Terra Commands
+
+The `Terra.it` commands have been removed. We noticed they encouraged anti-patterns while writing tests and made testing more confusing.
+
+These commands have been removed:
+
+- Terra.it.validatesElement
+- Terra.it.isAccessible
+- Terra.it.matchesScreenshot
+
+These commands can still be used but have had some changes:
+
+- Terra.validates.accessibility
+- Terra.validates.element
+- Terra.validates.screenshot
```

Comment:
Should we add a note or breaking change stating that `Terra.validates.screenshot` [no longer accepts an array of viewports for taking a screenshot in these viewports](https://github.com/cerner/terra-toolkit/pull/467#discussion_r517541669)?

Specifically, this is the [viewport option](https://github.com/cerner/terra-toolkit-boneyard/blob/main/src/wdio/services/TerraCommands/visual-regression.js#L69) we no longer support.

Reply:
I think so, I think that's important information for anyone uplifting:

Added: https://github.com/cerner/terra-toolkit/pull/574/commits/db541aac30859c6d51c91d56ff5f9989711ef3fe